scalikejdbc-mapper-generator-core/src/main/scala/scalikejdbc/mapper/CodeGenerator.scala

```diff
@@ -924,6 +981,12 @@ class CodeGenerator(table: Table, specifiedClassName: Option[String] = None)(imp
           |      val shouldBeNone = %className%.find(%primaryKeys%)
           |      shouldBeNone.isDefined should beFalse
           |    }
+          |    def batchInsert = this {
+          |      val list = %className%.findAll()
```

Comment:
No problem, I see.

And if 'entities' is good, I want to also change from 'list' to 'entities' in 'batchInsertMethod'.